test/io_service_tests.cpp

```diff
@@ -191,6 +191,10 @@ TEST_CASE("Timer cancellation"
 		}()));
 }
 
+// Disable this test under VS 2019.3
+// Results in ICE compiling the 'startTimer' lambda below.
+#if CPPCORO_COMPILER_MSVC != 192328105
```

Comment:
CC: @gornishanov I've just been trying to update cppcoro to work with MSVC 2019.3 and have started encountering some new compiler-crashes. I've disabled the problematic tests for now but wanted to give you a heads up.